modernomad/settings/common.py

```diff
@@ -98,6 +98,10 @@
 ANYMAIL = {
     "MAILGUN_API_KEY": env('MAILGUN_API_KEY', default=''),
 }
+ANYMAIL_SEND_DEFAULTS = {
+    'esp_extra': {'o:testmode': env('MAILGUN_TESTMODE', default='yes')}
+}
```

Comment:
Uses testmode by default, so if for whatever reason this environment variable is forgotten in staging it won't be a big deal.